docs/management/api/running-procs-api.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[running-procs-api]]
+=== Get processes
+
+Get processes on a host running {endpoint-cloud-sec}.
+
+You must have the `superuser` role to perform this action and at least an Enterprise license.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/endpoint/action/running_procs`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`endpoint_ids` |Array (String) |The IDs of each endpoint you want to issue this action to. |Yes
+|`alert_ids` |Array (String) |If this action is associated with any alerts, they can be specified here. The action will be logged in any cases associated with the specified alerts. |No
+|`case_ids` |Array (String) |Logs the action taken on specified cases. |No
```

Comment:
```suggestion
|`case_ids` |Array (String) |The ID(s) of case(s) where the action taken will be logged. |No
```

Reply:
If you accept this suggestion, it should also apply to the other places where `case_ids` appears in this PR